webui-page/webui.js

```diff
@@ -618,8 +611,15 @@ function audioPause() {
   }
 }
 
-function setupNotification() {
+function setupNotification(json) {
   if ('mediaSession' in navigator) {
+    if (typeof navigator.mediaSession.setPositionState === "function") {
```

Comment:
the common idioms is to test directly like `if (navigator.mediaSession.setPositionState) {`

this would apply to the mediaSession tests too like `if (navigator.mediaSession) {`, but i was leaving existing code alone rather than rewriting things for no practical gain

Reply:
My question was about this line. I wondered if there is some more elegant way to check for support for this, rather than just checking if it's defined. But I'm ok with what you suggested.